src/pages/Landing.tsx

```diff
@@ -0,0 +1,43 @@
+import { FormControl, TextField, Grid, GridList } from '@material-ui/core'
+import { useAppDispatch } from '../app/hooks'
+import { createMeeting } from './meetingNameSlice'
+import { useNavigate } from 'react-router-dom'
+import '../styles/button.css'
+
+function Landing(){
+  const dispatch = useAppDispatch();
+  const navigate = useNavigate();
+
+  return(
+    <>
+      <Grid 
+        container
+        spacing={0}
+        direction="column"
+        alignItems="center"
+        justify="center"
+        style={{ minHeight: "100vh" }}
+      >
+  
+      <FormControl>
+        <TextField
+          fullWidth 
+          placeholder="Name the Meeting:"
+          aria-label="Name the Meeting:"
+          onChange={(event)=>dispatch(createMeeting(event.target.value))}
```

Comment:
I think here you dispatch a createMeeting action every time the text field changes: 
![image](https://user-images.githubusercontent.com/32407086/159166628-0fd155da-f171-4505-bbc3-9e7c670b26c2.png)

Maybe instead we should dispatch a createMeeting action only when the Create button is clicked?

Reply:
I would suggest for the meeting name on the landing page to be a React state (`useState("")`), and to dispatch a redux action when create is pressed (rename it to `setMeetingName()` maybe?)